### PR TITLE
add project assignments chart screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create Project Assignments Chart Screen #274](https://github.com/nerubia/kh360-react/issues/274)

BE PR: [update project member api for proj assignment screen #333](https://github.com/nerubia/kh360-nodejs/pull/333)

### CODE REVIEW

File: src/components/ui/gantt-chart/project-column.tsx
Comment (jatnerubia) [inline src/components/ui/gantt-chart/project-column.tsx:1]: why is it in the UI folder?

File: src/components/ui/gantt-chart/project-header.tsx
Comment (jatnerubia) [inline src/components/ui/gantt-chart/project-header.tsx:1]: why is it in the UI folder?

File: src/components/ui/gantt-chart/tooltip/project-tooltip.tsx
Comment (jatnerubia) [inline src/components/ui/gantt-chart/tooltip/project-tooltip.tsx:1]: use tooltip

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**